kfs-cam/src/main/java/org/kuali/kfs/module/cam/businessobject/AssetBase.java

```diff
@@ -2170,4 +2172,19 @@ public String getCamsComplexMaintenanceDocumentLookup() {
         return managedLists;
     }
 
+    public List<Note> getBoNotes() {
+        if (!StringUtils.isEmpty(getObjectId())) {
+            boNotes = SpringContext.getBean(NoteService.class).getByRemoteObjectId(getObjectId());

```

Comment:
As far as I can tell from the coding standards with respect to injections, SpringContext should be the proper way to access this.